src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -1415,12 +1415,9 @@ public void onPlayerSlotChange(PlayerItemHeldEvent event) {
 		BendingPlayer bPlayer = BendingPlayer.getBendingPlayer(player);
 		int slot = event.getNewSlot() + 1;
 		
-		if (bPlayer.getAbilities().get(slot) != null ) {

```

Comment:
Yes, I have no idea why that was ever null but Matan posted a NPE that was thrown because of it